internal/v2/repository/iam/aggregate.go

```diff
@@ -2,6 +2,18 @@ package iam
 
 import (
 	"context"
+	"github.com/caos/zitadel/internal/v2/repository/iam/policy/label"
+	iam_login "github.com/caos/zitadel/internal/v2/repository/iam/policy/login"
+	"github.com/caos/zitadel/internal/v2/repository/iam/policy/login/idpprovider"
```

Comment:
maybe same as below. it might make sense to rename the package to provider and then `IDP` as prefix of the structs